go.mod

```diff
@@ -1,207 +1,216 @@
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsecscontainermetricsreceiver v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsxrayreceiver v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/statsdreceiver v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.98.0
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.4
-	go.opentelemetry.io/collector/component v0.94.1
-	go.opentelemetry.io/collector/confmap v0.94.1
-	go.opentelemetry.io/collector/exporter v0.94.1
-	go.opentelemetry.io/collector/exporter/loggingexporter v0.94.1
-	go.opentelemetry.io/collector/exporter/otlpexporter v0.94.1
-	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.94.1
-	go.opentelemetry.io/collector/extension v0.94.1
-	go.opentelemetry.io/collector/extension/ballastextension v0.94.1
-	go.opentelemetry.io/collector/extension/zpagesextension v0.94.1
-	go.opentelemetry.io/collector/featuregate v1.1.0
-	go.opentelemetry.io/collector/otelcol v0.94.1
-	go.opentelemetry.io/collector/processor v0.94.1
-	go.opentelemetry.io/collector/processor/batchprocessor v0.94.1
-	go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.94.1
-	go.opentelemetry.io/collector/receiver v0.94.1
-	go.opentelemetry.io/collector/receiver/otlpreceiver v0.94.1
+	github.com/stretchr/testify v1.9.0
+	go.opentelemetry.io/collector/component v0.98.0
+	go.opentelemetry.io/collector/confmap v0.98.0
+	go.opentelemetry.io/collector/confmap/converter/expandconverter v0.98.0
+	go.opentelemetry.io/collector/confmap/provider/envprovider v0.98.0
+	go.opentelemetry.io/collector/confmap/provider/fileprovider v0.98.0
+	go.opentelemetry.io/collector/confmap/provider/httpprovider v0.98.0
+	go.opentelemetry.io/collector/confmap/provider/httpsprovider v0.98.0
+	go.opentelemetry.io/collector/confmap/provider/yamlprovider v0.98.0
+	go.opentelemetry.io/collector/exporter v0.98.0
+	go.opentelemetry.io/collector/exporter/loggingexporter v0.98.0
+	go.opentelemetry.io/collector/exporter/otlpexporter v0.98.0
+	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.98.0
+	go.opentelemetry.io/collector/extension v0.98.0
+	go.opentelemetry.io/collector/extension/ballastextension v0.98.0
+	go.opentelemetry.io/collector/extension/zpagesextension v0.98.0
+	go.opentelemetry.io/collector/featuregate v1.6.0
```

Comment:
Do the stable module versions need to match the pre 1.0 versions that they were released with? For example, [v0.98.0 and v1.5.0 were released together.](https://github.com/open-telemetry/opentelemetry-collector/blob/release/v0.98.x/versions.yaml)

Reply:
Yes versions should match there as well, committed the changes.